cmd/convert.go

```diff
@@ -44,6 +45,12 @@ can be converted into a 'konnect' configuration file.`,
 			if err != nil {
 				return fmt.Errorf("converting file: %v", err)
 			}
+			if convertCmdDestinationFormat == "konnect" {
+				cprint.UpdatePrintf("Notice: The `konnect` format type has been deprecated" +
+					"as of v1.12.\nPlease use the `kong-gateway` format with `deck <cmd>` " +
+					"if you would like \nto declaratively manage your Kong gateway config in Konnect.\n" +
+					"See https://docs.konghq.com/konnect/deployment/import/\n")
```

Comment:
Warning: `konnect` format type was deprecated in v1.12 and it will be removed in a future version. Please use your Kong configuration files with `deck <cmd>`. Please see https://docs.konghq.com/konnect/deployment/import.

With appropriate `\n`s.